projects/packages/videopress/src/client/admin/components/videos-grid/style.module.scss

```
@@ -0,0 +1,6 @@
+.wrapper {
+	--videopress-video-item-width: 360px;
+	--videopress-videos-grid: calc( var( --videopress-video-item-width ) * 3 + 24px * 2 ); // 3 columns - max width.
```

I don't know if it would be better, as the line is long, but that `24px` is spacing. Maybe isolating it on a variable would make it easier to read.
```suggestion
	--videopress-video-item-gap: calc( var( --spacing-base ) * 3 ); // 24px;
	--videopress-videos-grid: calc( var( --videopress-video-item-width ) * 3 + var( --videopress-video-item-gap ) * 2 ); // 3 columns - max width.
```

---

There is an empty `.scss` file that I can't comment on as well.

---

removing...